modules/templates/helper.go

```diff
@@ -278,6 +281,36 @@ func renderCommitMessage(msg string, opts markup.RenderIssueIndexPatternOptions)
 	return template.HTML(msgLines[0])
 }
 
+func RenderCommitBody(msg, urlPrefix string, metas map[string]string) template.HTML {
+	return renderCommitBody(msg, markup.RenderIssueIndexPatternOptions{
+		URLPrefix: 	urlPrefix,
+		Metas:		metas,
+	})
+}
+
+func RenderCommitBodyLink(msg, urlPrefix string, urlDefault string, metas map[string]string) template.HTML {
+	return renderCommitMessage(msg, markup.RenderIssueIndexPatternOptions{
+		DefaultURL: urlDefault,
+		URLPrefix:  urlPrefix,
+		Metas:      metas,
+	})
+}
+
+func renderCommitBody(msg string, opts markup.RenderIssueIndexPatternOptions) template.HTML {
+	cleanMsg := template.HTMLEscapeString(msg)
+	fullMessage := string(markup.RenderIssueIndexPattern([]byte(cleanMsg), opts))
+	msgLines := strings.Replace(strings.TrimSpace(fullMessage), "\n\n", "\n", -1)
+	body := strings.Split(msgLines,"\n")
+	if len(body) == 0 {
+		return template.HTML("")
+	}
+	return template.HTML(strings.Join(body[1:], "\n"))
```

Comment:
I think we should differentiate between single and double new-lines, since a single newline may just be wrapping a long line, but a double newline typically indicates the start of a new paragraph. In fact, I think the best solution is just to render newlines as they appear in the commit message (this is what Github does).

Also, note that newlines in a `template.HTML` will *not* be rendered as newlines. The somewhat hacky way to fix this problem is to replace newlines with `<br>`.

Reply:
Yeah, I initially tried joining on `<br>` but then I looked at Github and saw that they wrapped their commit messages in `<pre>`. However, if that's the case I think the method needs to work differently because it already splits on `\n` and then I discard the first element and join the others. It seems to render correctly, but I'll see what I can figure out.